src/test/java/org/broadinstitute/dsm/GBFTest.java

```diff
@@ -168,25 +169,30 @@ public void inputConfirmationGBFKit() throws Exception {
         String apiKey = "";
         if (apiKey != null) {
             long testDate = 1609172139000L; //change that to the date of testing!
-            long start = 0;
+            long start = 1608613200000L;
             long end = testDate;
 
             JSONObject payload = new JSONObject().put("startDate", SystemUtil.getDateFormatted(start)).put("endDate", SystemUtil.getDateFormatted(end));
             String sendRequest = GBF_URL + GBFRequestUtil.CONFIRM_ENDPOINT;
             Response gbfResponse = GBFRequestUtil.executePost(Response.class, sendRequest, payload.toString(), apiKey);
 
             GBFRequestUtil gbf = new GBFRequestUtil();
-                String query = "SELECT * " +
-                        "FROM ddp_kit_request req  " +
-                        "LEFT JOIN ddp_kit kit ON  " +
-                        "(req.dsm_kit_request_id = kit.dsm_kit_request_id) " +
-                        "WHERE " +
-                        "req.ddp_instance_id = ?  " +
-                        "AND external_order_status = 'SHIPPED' " +
-                        "AND external_response is null " +
-                        "ORDER BY external_order_date ASC ";
+            String query = "SELECT * " +
+                    "FROM ddp_kit_request req  " +
+                    "LEFT JOIN ddp_kit kit ON (req.dsm_kit_request_id = kit.dsm_kit_request_id)  " +
+                    "LEFT JOIN ddp_participant_exit ex ON (ex.ddp_instance_id = req.ddp_instance_id AND ex.ddp_participant_id = req.ddp_participant_id)   " +
```

Comment:
ddp_participant_exit can be removed if not used

Reply:
ok